libraries/mongodb.rb

```diff
@@ -84,13 +93,14 @@ def self.configure_replicaset(node, name, members)
       return
     end
 
+    Chef::Log.info("#{members.map(&:fqdn).join(', ')}")
```

Comment:
Please update this to explain what is being dumped to avoid confusion when reviewing the output of a full Chef run

Reply:
Oh I'll actually delete that, I don't think it's needed.